bb

```diff
@@ -1389,5 +1409,6 @@ case "$cmd" in
 	pay) bb_pay $@;;
 	grades) bb_grades "$@";;
 	materials) bb_materials "$@";;
+	announcements) bb_announcements "$@";;
```

Comment:
This adds announcements, not materials - I think git gets the diff right, but GitHub seems to get it wrong. It may be good to merge this on the command line just in case.